### PR TITLE
Reduce liquid volume per turn by a factor of 6

### PR DESCRIPTION
**Pull request by ZhilkinSerg**
#### Summary

```SUMMARY: Bugfixes "Reduce liquid volume per turn by a factor of 6"```

#### Purpose of change

Amount of liquid transferred per turn was not changed after we moved from 6 to 1 second turns.

#### Describe the solution

Reduced amount of liquid transferred per turn by a factor of 6.

### CODE REVIEW

File: src/activity_handlers.cpp [hunk 1]
```diff
@@ -1330,7 +1330,7 @@ void activity_handlers::fill_liquid_do_turn( player_activity *act, player *p )
                 break;
         }
 
-        static const auto volume_per_turn = units::from_liter( 4 );
+        static const auto volume_per_turn = units::from_liter( 4 / 6 );
```
Comment (KorGgenT) [inline src/activity_handlers.cpp]: while you're at it, wanna remove that auto in favor of units::volume?

Comment (ZhilkinSerg) [inline src/activity_handlers.cpp]:
```suggestion
        static const units::volume volume_per_turn = units::from_liter( 4 / 6 );
```

File: src/activity_handlers.cpp [hunk 2]
```diff
@@ -1330,7 +1330,7 @@ void activity_handlers::fill_liquid_do_turn( player_activity *act, player *p )
                 break;
         }
 
-        static const auto volume_per_turn = units::from_liter( 4 );
+        static const units::volume volume_per_turn = units::from_liter( 4 / 6 );
         const int charges_per_turn = std::max( 1, liquid.charges_per_volume( volume_per_turn ) );
         liquid.charges = std::min( charges_per_turn, liquid.charges );
```
Comment (ZhilkinSerg) [inline src/activity_handlers.cpp]:
```suggestion
        liquid.charges = std::min( charges_per_second, liquid.charges );
```

File: src/activity_handlers.cpp [hunk 3]
```diff
@@ -1330,7 +1330,7 @@ void activity_handlers::fill_liquid_do_turn( player_activity *act, player *p )
                 break;
         }
 
-        static const auto volume_per_turn = units::from_liter( 4 );
+        static const units::volume volume_per_turn = units::from_liter( 4 / 6 );
         const int charges_per_turn = std::max( 1, liquid.charges_per_volume( volume_per_turn ) );
```
Comment (ZhilkinSerg) [inline src/activity_handlers.cpp]:
```suggestion
        const int charges_per_second = std::max( 1, liquid.charges_per_volume( volume_per_second ) );
```

File: src/activity_handlers.cpp [hunk 4]
```diff
@@ -1330,7 +1330,7 @@ void activity_handlers::fill_liquid_do_turn( player_activity *act, player *p )
                 break;
         }
 
-        static const auto volume_per_turn = units::from_liter( 4 );
+        static const units::volume volume_per_turn = units::from_liter( 4 / 6 );
```
Comment (ZhilkinSerg) [inline src/activity_handlers.cpp]:
```suggestion
        static const units::volume volume_per_second = units::from_liter( 4 / 6 );
```

File: src/activity_handlers.cpp [hunk 5]
```diff
@@ -1330,9 +1330,9 @@ void activity_handlers::fill_liquid_do_turn( player_activity *act, player *p )
                 break;
         }
 
-        static const auto volume_per_turn = units::from_liter( 4 );
-        const int charges_per_turn = std::max( 1, liquid.charges_per_volume( volume_per_turn ) );
-        liquid.charges = std::min( charges_per_turn, liquid.charges );
+        static const units::volume volume_per_second = units::from_liter( 4 / 6 );
+         const int charges_per_second = std::max( 1, liquid.charges_per_volume( volume_per_second ) );
```
Comment (ZhilkinSerg) [inline src/activity_handlers.cpp]:
```suggestion
        const int charges_per_second = std::max( 1, liquid.charges_per_volume( volume_per_second ) );
```

File: src/activity_handlers.cpp [hunk 6]
```diff
@@ -1330,9 +1330,9 @@ void activity_handlers::fill_liquid_do_turn( player_activity *act, player *p )
...
+         liquid.charges = std::min( charges_per_second, liquid.charges );
```
Comment (ZhilkinSerg) [inline src/activity_handlers.cpp]:
```suggestion
        liquid.charges = std::min( charges_per_second, liquid.charges );
```

File: src/activity_handlers.cpp [hunk 7]
```diff
@@ -1330,9 +1330,9 @@ void activity_handlers::fill_liquid_do_turn( player_activity *act, player *p )
                 break;
         }
 
-        static const auto volume_per_turn = units::from_liter( 4 );
-        const int charges_per_turn = std::max( 1, liquid.charges_per_volume( volume_per_turn ) );
-        liquid.charges = std::min( charges_per_turn, liquid.charges );
+        static const units::volume volume_per_second = units::from_liter( 4 / 6 );
```
Comment (BevapDin) [inline src/activity_handlers.cpp]: You changed this to 1 liter per second. `4 / 6` is `1` in C++ (both operands are integers, so integer division is performed). If you wan't floating points, you have to explicitly make one operand a floating point value: `( 4. / 6 )`

Comment (AMurkin) [reply src/activity_handlers.cpp]: It's 0 actually.

Comment (kevingranade) [reply src/activity_handlers.cpp]: Ahhh my brain, I should have noticed that but for some reason I thought it was ok because of the units interface.

**Pull request merged**